### ISSUE TITLE
Error on message application with no cover_image or icon

### ISSUE THREAD

**Issue by nathanwfranke** [DV8FromTheWorld/JDA#978]
[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

  
- [ ] I have checked for similar issues.
- [ ] I have updated to the [latest JDA version][download].
- [ ] I have checked the branches or the maintainers' PRs for upcoming features/bug fixes.

### Issue Type

- [x] Bug Report
- [ ] Feature Request

### Description

Stacktrace:

```
[JDA MainWS-ReadThread] WARN WebSocketClient - Got an unexpected Json-parse error. Please redirect following message to the devs:
        JSONObject["cover_image"] not found.
        MESSAGE_CREATE -> // My message JSON was here, removed for privacy
org.json.JSONException: JSONObject["cover_image"] not found.
        at org.json.JSONObject.get(JSONObject.java:471)
        at org.json.JSONObject.getString(JSONObject.java:717)
        at net.dv8tion.jda.core.entities.EntityBuilder.createMessage(EntityBuilder.java:786)
        at net.dv8tion.jda.core.entities.EntityBuilder.createMessage(EntityBuilder.java:750)
        at net.dv8tion.jda.core.handle.MessageCreateHandler.handleInternally(MessageCreateHandler.java:61)
        at net.dv8tion.jda.core.handle.SocketHandler.handle(SocketHandler.java:37)
        at net.dv8tion.jda.core.requests.WebSocketClient.onDispatch(WebSocketClient.java:868)
        at net.dv8tion.jda.core.requests.WebSocketClient.onEvent(WebSocketClient.java:766)
        at net.dv8tion.jda.core.requests.WebSocketClient.handleEvent(WebSocketClient.java:745)
        at net.dv8tion.jda.core.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:903)
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:368)
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:270)
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:990)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:749)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
```

![image](https://i.imgur.com/w7Kv4Mi.png)

**Comment by Sanduhr32** [comment]
Fixed in 3.8.3_462. My bad for making it not nullable when it was released because they omitted it because a certain other value wasn't set.

**Comment by MinnDevelopment** [comment]
Fixed in [3.8.3_462](https://github.com/DV8FromTheWorld/JDA/commit/06f7823033814fe8455c6dfcb9c5274c93328dab)
